extmod/network_cyw43.c

```diff
@@ -536,6 +536,11 @@ STATIC const mp_rom_map_elem_t network_cyw43_locals_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR_config), MP_ROM_PTR(&network_cyw43_config_obj) },
 
     // Class constants.
+    { MP_ROM_QSTR(MP_QSTR_STA_IF), MP_ROM_INT(MOD_NETWORK_STA_IF) },
+    { MP_ROM_QSTR(MP_QSTR_AP_IF), MP_ROM_INT(MOD_NETWORK_AP_IF) },
+    { MP_ROM_QSTR(MP_QSTR_AUTH_OPEN), MP_ROM_INT(CYW43_AUTH_OPEN) },
+    { MP_ROM_QSTR(MP_QSTR_AUTH_WPA_PSK), MP_ROM_INT(CYW43_AUTH_WPA2_AES_PSK) },
```

Comment:
I suggest using `SEC_WPA_WPA2` for this constant, because:
- It should explicitly state that it's mixed WPA/WPA2 mode.
- The "psk" (pre shared key) suffix isn't really needed.  It's only to distinguish with other auth modes like enterprise.  But enterprise can be distinguished with a `_ENT` suffix.

Also, the corresponding cyw43 constant should be `CYW43_AUTH_WPA2_MIXED_PSK`.

Can add `SEC_WPA2` with `CYW43_AUTH_WPA2_AES_PSK` if needed.

Reply:
@dpgeorge So remove `_PSK` from any other WPA constant as well ? Like :
```
    { MP_ROM_QSTR(MP_QSTR_SEK_WPA_PSK), MP_ROM_INT(AUTH_WPA_PSK) },
    { MP_ROM_QSTR(MP_QSTR_SEK_WPA2_PSK), MP_ROM_INT(AUTH_WPA2_PSK) },
    { MP_ROM_QSTR(MP_QSTR_SEK_WPA_WPA2_PSK), MP_ROM_INT(AUTH_WPA_WPA2_PSK) },
```